### PR TITLE
Create firebase (upload data to and retrieve data from firebase)

### PR DESCRIPTION
**Pull request by aerinpark**
Features Added:
- Selecting an image from the Gallery
- Uploading slide image and its data into Firebase
- Retrieve slide images and data from Firebase

### CODE REVIEW

File: app/src/main/res/layout/slides_card.xml [hunk 1]
```diff
@@ -81,7 +64,100 @@
...
+<!--    app:cardCornerRadius="4dp"-->
+<!--    app:cardElevation="3dp">-->
+
+<!--    <RelativeLayout-->
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="wrap_content"-->
+<!--        android:background="@color/white">-->
+
+<!--        <ImageView-->
+<!--            android:id="@+id/slide_card_image"-->
```
Comment (Jramacula) [inline app/src/main/res/layout/slides_card.xml:89]: Commented code we can remove

Comment (Jramacula) [inline app/src/main/res/layout/slides_card.xml:89]: Commented Code we can remove

**Review by Jramacula — APPROVED** [review]

**Review by bperdija — CHANGES_REQUESTED** [review]
Will look at:
- Naming properly in Firebase
- Providing warning on splash 
- Possible general UI tidying

**Review by bperdija — APPROVED** [review]

**Comment by bperdija** [comment]
Realized naming in firebase shouldn't matter to the overall implementation (the naming scheme is meant to prevent duplicates). We *should* be able to search via other methods.

**Pull request merged**